modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test/src/testIntegration/java/com/liferay/site/initializer/extender/internal/test/BundleSiteInitializerTest.java

```diff
@@ -634,6 +637,34 @@ private void _assertAssetCategories() throws Exception {
 			testAssetCategory4.getExternalReferenceCode());
 	}
 
+	private void _assertAssetCategories1() throws Exception {
+		Company company = _companyLocalService.getCompany(
+			_serviceContext.getCompanyId());
+
+		AssetVocabulary assetVocabulary =
+			_assetVocabularyLocalService.getGroupVocabulary(
+				company.getGroupId(), "Test Asset Vocabulary 1");
+
+		AssetCategory assetCategory = _assetCategoryLocalService.fetchCategory(
+			company.getGroupId(),
+			AssetCategoryConstants.DEFAULT_PARENT_CATEGORY_ID,
+			"Test Asset Category 1", assetVocabulary.getVocabularyId());
+
+		Assert.assertNotNull(assetCategory);
+
+		Role role = _roleLocalService.getRole(
+			company.getCompanyId(), RoleConstants.USER);
+
+		boolean hasUserViewPermission =
+			_resourcePermissionLocalService.hasResourcePermission(
+				company.getCompanyId(), assetCategory.getModelClassName(),
+				ResourceConstants.SCOPE_INDIVIDUAL,
+				String.valueOf(assetCategory.getCategoryId()), role.getRoleId(),
+				ActionKeys.VIEW);
+
+		Assert.assertTrue(hasUserViewPermission);
+	}
```

Comment:
We don't need to add a new test for categories since we [already have a test for that](https://github.com/liferay/liferay-portal/blob/1da40b0328ae5ca1f3dab9b0701a465547d5e402/modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test/src/testIntegration/java/com/liferay/site/initializer/extender/internal/test/BundleSiteInitializerTest.java#L594)

Since we are only changing the replace, I think we can change the [catalog name check](https://github.com/liferay/liferay-portal/blob/1da40b0328ae5ca1f3dab9b0701a465547d5e402/modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test/src/testIntegration/java/com/liferay/site/initializer/extender/internal/test/BundleSiteInitializerTest.java#L870) to assert if the replace worked as expected